Packs/FeedBambenekConsulting/Integrations/FeedBambenekConsulting/FeedBambenekConsulting.yml

```diff
@@ -78,6 +69,24 @@ configuration:
   name: proxy
   required: false
   type: 8
+- display: Sub-Feeds
+  name: url
+  options:
+  - http://osint.bambenekconsulting.com/feeds/c2-ipmasterlist.txt
+  - http://osint.bambenekconsulting.com/feeds/c2-ipmasterlist-high.txt
+  - http://osint.bambenekconsulting.com/feeds/c2-dommasterlist.txt
+  - http://osint.bambenekconsulting.com/feeds/c2-dommasterlist-high.txt
+  - https://faf.bambenekconsulting.com/feeds/dga-feed.gz
+  - https://faf.bambenekconsulting.com/feeds/dga-feed-high.gz
+  - https://faf.bambenekconsulting.com/feeds/dga/c2-masterlist.txt
+  - https://faf.bambenekconsulting.com/feeds/dga/c2-masterlist-high.txt
+  - https://faf.bambenekconsulting.com/feeds/sinkhole/latest.csv
```

Comment:
maybe we should take the names from their site, please check this option
https://osint.bambenekconsulting.com/feeds/